backtracking/n-queen.dart

```diff
@@ -0,0 +1,119 @@
+import 'dart:io';
+import 'package:test/test.dart';
+
+/*N-Queens Problem
+
+N - Queens problem is to place n - queens in such a manner 
+on an n x n chessboard that no queens attack each other by being in the same row, column or diagonal
+We have to find maximum no of solutions that are possible.
+
+.*/
+
+// board
+List<List<int>> board = [];
+
+// checking is it safe place for a queen.
+bool isSafe(int n, int x, int y) {
```

Comment:
can you use descriptive variable names instead of x, y, n etc

Reply:
Sure I will  change